package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "test-project",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.6.0",
+    "@material-ui/icons": "^4.5.1",
+    "axios": "^1.2.0",
+    "chat-client": "^1.0.3",
+    "history": "^4.10.1",
+    "node-sass": "^4.13.0",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "react-material-ui-form-validator": "^2.0.9",
+    "react-redux": "^7.1.3",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-router-redux": "^4.0.8",
+    "react-scripts": "3.2.0",
+    "react-swipeable-views": "^0.14.0",
+    "react-swipeable-views-utils": "^0.14.0",
+    "redux": "^4.0.4",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
I think there is no need in this dependency on production side. 
But if you use this dependency on dev you can create "devDependencies" field in this file and add "redux-devtools-extension" to that object